server/src/test/java/com/ca/lsp/cobol/usecases/TestExtraneousInputEOFExpecting.java

```diff
@@ -37,7 +37,7 @@
   private static final String MESSAGE =
       "Extraneous input '<EOF>' expected {ACCEPT, ADD, ALTER, CALL, CANCEL, CLOSE, "
           + "COMPUTE, CONTINUE, DELETE, DISABLE, DISPLAY, DIVIDE, ENABLE, ENTRY, "
-          + "EVALUATE, EXEC, EXHIBIT, EXIT, GENERATE, GOBACK, GO, IF, INITIALIZE, "
+          + "EVALUATE, EXEC, EXHIBIT, EXIT, GENERATE, GO, GOBACK, IF, INITIALIZE, "
```

Comment:
Is it necessary to provide sorted expected values ?

Reply:
They are not really sorted, it is the order of the rules in the lexer grammar. And the best practice to keep them sorted when it is possible